config.toml

```diff
@@ -19,7 +19,15 @@ bestqualityfirst = true
 # EXPERIMENTAL enable temporal layer change is currently an experimental feature,
 # enable only for testing.
 enabletemporallayer = false
-
+# Sets the audio level volume threshold.
+# Values from [0-127] where 0 is the loudest.
+# Audio levels are read from rtp extension header according to:
+# https://tools.ietf.org/html/rfc6464
+audiolevelthreshold = 40
+# Sets the interval in which the SFU will emit the audio level
+# events in [ms]
+# Values 500-5000
```

Comment:
Why limit the interval to 500-5000? I could easily see a case for <500ms, 300 and even 100 seem reasonable values. On the other end I can't see why anyone would want >5s, but why limit it, probably some strange use cases out there.

Reply:
We can lower or remove the limits, Just not sure if something less than 25 sample would be useful for something.

So 50-5000 is better or just remove it?

Reply:
Ideally the interval at which we send the "X is speaking" messages wouldn't affect the interval over which we calculate if someone is speaking or not.

I vote just remove it. Maybe if it's <100ms (or 300ms?) log a warning, just to cut down on "server is slow" bug reports?

Reply:
well with this implementation it is affected, since the values are resetted on timer, since I dont want to keep more buffers to prevent many allocations.